docs/source/noaa_csp.rst

```diff
@@ -40,160 +51,252 @@ Login to the NOAA CSP
 
 Log in to the `NOAA CSP <http://noaa.parallel.works/login>`_ and into
 the resources configuration. The user should arrive at the following
-screen.
+screen. Click the "blue" box pointed by the red arrow to login.
+
+.. image:: _static/noaacsp_login_1.png
 
-Note that the ``Username or email`` query is case-sensitive. The user
-will then be prompted for their respective RSA token key using the
-same application use for the other RDHPCS machines (i.e., Hera, Jet,
-etc.,).
+Fill the ``Username / Email`` box with your own username or NOAA-email.
+Note that the ``Username or email`` query is case-sensitive.
+And then the respective RSA token key using the same application used
+for the other RDHPCS machines (i.e., Hera, Jet, etc.,).
 
-.. image:: _static/noaacsp_login.png
+.. image:: _static/noaacsp_login_2.png
 
 *******************************
 Configure the NOAA CSP Instance
 *******************************
 
-Once logged into the NOAA CSP, navigate to the :red-text:`RESOURCES` section
-and click the ``+ Add Resource`` button in the upper-right corner as
-illustrated below.
+Once logged into the NOAA CSP, navigate to the :red-text:`Marketplace` section
+at the left panel as pointed by the red arrow, and click.
+Scrow down to selecet "AWS EPIC Wei CentOS" as in red-circle.
+Note, current global-workflow still using CentOS built spack-stack.
+It will be updated to Rocky 8 soon.
 
 .. image:: _static/noaacsp_instance_1.png
    
-Next, the mandatory attributes for the respective instance must be
-defined as shown in the illustration below.
+Next, click "Forklatest" as shown in the red-circle.
 
 .. image:: _static/noaacsp_instance_2.png
-
-The annotated attributes and their respective descriptions are as
-follows.
-
-1. A unique name for the instance. Best practices suggest one that is
-   clear, concise, and relevant to the application.
-2. A short description of the instance, i.e., ``This instance supports
-   this <task name> task.``
-3. Tag(s) describing and identifying the respective instance. These
-   allow for improved bookkeeping, especially when a user has multiple
-   or concurrent instance types.
-
-Next, the cluster is defined as shown in the following illustration.
+   
+Please provide a unique name (box pointer by the red-arrow) for the instance.
+Best practices suggest one that is clear, concise, and relevant to the application.
+Click :red-text:`Fork` (in the red-circle) to fork an instance.
```

Comment:
Same as my comment above -- `:red-text:` won't render in red.

Reply:
I'll use double single quota instead of red:text.